src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -55,6 +55,8 @@
       - firewalld
       - gdb
       - gdb-gdbserver
+      - iproute
+      - iproute-tc
```

Comment:
This has to go to minimal package set since it is test dependency. So it needs to go to `Install systemd and minimal set of packages`.

We could also add it only to `Install test dependencies on client` but this would make it available only on client host. But I think this tool may be handy on other hosts as well.